CHANGELOG.md

```diff
@@ -4,6 +4,168 @@
 
 ### New features
 
+#### Events V2
```

Comment:
Should we add a section also about how workqueues can now be configured?

Certificate variables have also changed and now have much moredata in them I think. 

Configurable overview page, `analytics: boolean` flag and Metabase dataset change overall (might be better suited for country config).
Also the trigger hooks we added for core to call CC on every action might be good to mention.

Reply:
The workqueues are mentioned here: https://github.com/opencrvs/opencrvs-core/pull/10895/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR138

Event Overview: https://github.com/opencrvs/opencrvs-core/pull/10895/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR150

Metabase: We can mention that metabase configuration has been fully moved to countryconfig

Trigger hooks: This section can be expanded a bit in that case: https://github.com/opencrvs/opencrvs-core/pull/10895/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR128

Certificate variable: missing currently